oe_whitelabel.theme

```diff
@@ -184,3 +184,53 @@ function oe_whitelabel_preprocess_media__document__default(&$variables) {
     $variables['translations'][] = $wrapper->toFileValueObject();
   }
 }
+
+/**
+ * Implements hook_element_info_alter().
+ */
+function oe_whitelabel_element_info_alter(array &$info): void {
+  $module_handler = \Drupal::service('module_handler');
```

Comment:
We could also create a submodule that depends on slim_select make the thing thus optional and drop this if. The risk of this proposal would be that hook_library_info_alter here would be executed before slim_select's